src/classes/UTIL_SObject.cls

```diff
@@ -68,7 +68,9 @@ public with sharing class UTIL_SObject {
     * @return Set<Id>
     */
     public static Set<Id> extractId(List<SObject> records, Schema.SObjectField field) {
-        return new Set<Id>((List<Id>) extractField(List<Id>.Class, records, field.getDescribe().getName()));
+        Set<Id> values = new Set<Id>((List<Id>) extractField(List<Id>.Class, records, field.getDescribe().getName()));
```

Comment:
@force2b , can you please review this change so utility does not return null Ids?
@ravensisco , affected is currency on Opp verification: AC2. There is no change in behaviour, but when Opps are not assigned to a RD, there is no redundant DB query to get RD Ids.

Reply:
The change makes sense. I can see a null value in this set causing issues.